extensions/undertow-websockets/runtime/src/main/java/io/quarkus/undertow/websockets/runtime/WebsocketConfig.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.undertow.websockets.runtime;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot
```

Comment:
The file can be moved to deployment module?

Reply:
oops, forgot to mark it as runtime configurable.

Reply:
I meant to say that since the configuration is only used during build time maybe we could have moved it to the deployment module all together?

Reply:
I thought I could move it to runtime, but it caused problems as this is done in static init, so I ended up moving it to deployment time.

Reply:
Thanks.